UIB webhook-template documentation (#364)
TheThingsIndustries/lorawan-stack-docs: doc/content/integrations/cloud-integrations/uib/_index.md

---
title: "UIB"
description: ""
weight: 

> aliases: ["/integrations/cloud-integrations/uib/tts-setup"]

Reviewer: There's no alias needed when this page isn't moved. We use aliases to make sure that old links are still working.

---

[UIB's](https://www.uib.ai/) technology makes human to machine communications simple.

<!--more-->

## Prerequisites

To send a message via WhatsApp, you need to have a WhatsApp Business Account and the content of the notification message which you need to send to the customer has to be submitted to WhatsApp and approved, before it can be used.

> Please send us an email to support@uib.ai, with the company details, Facebook Business Manager ID and an approximate no. of messages you want to send per month. Our support team will revert back with the pricing and next steps.

Reviewer: ```suggestion
Please send UIB an email to [support@uib.ai](mailto:support@uib.ai), with your company details, Facebook Business Manager ID and an approximate no. of messages you want to send per month. UIB's support team will revert back with the pricing and next steps.
```

Please note that the template/notification name, name space, access key etc will be provided once the notification message content is approved by WhatsApp.

## Configure {{% tts %}}

> In order for UIB to be able to decode the data coming from {{% tts %}}, you need to create an uplink [payload formatter]({{< ref "/integrations/payload-formatters" >}}) on {{% tts %}} in order to decode the uplink payload and set fields in the `decoded_payload` object of the uplink message. The notification message will be sent to user if the `decoded_payload_warnings` is not empty. You can push a text to the warnings array according to your parameter thresholds(field1,field2 etc can be the data of your device, like temprature or pressure). The example uplink payload formatter is shown below.

Reviewer: ```suggestion
In order for UIB to be able to decode the data coming from {{% tts %}}, you need to create an uplink [payload formatter]({{< ref "/integrations/payload-formatters" >}}) on {{% tts %}} in order to decode the uplink payload and set fields in the `decoded_payload` object of the uplink message. The notification message will be sent to user if the `decoded_payload_warnings` is not empty. You can push a text to the warnings array according to your parameter thresholds (`field1`, `field2` etc can be the data of your device, like temperature or pressure). The example uplink payload formatter is shown below.
```
